### ISSUE TITLE
How to map REST API response with dynamic properties to output claims

### ISSUE THREAD

**Issue by svirchenko** [azure-ad-b2c/samples#501]
Hi

I am trying to map REST API response with dynamic properties to output claims. Unfortunately, I can't find a way to do it in official documentation.

**Dynamic properties** of REST API:
- extension_ClientId_prop1
- extension_ClientId_prop2
- extension_ClientId_propN

where **ClientId** is a Guid identifier _without hyphen_ (for example, extension_7b230031ff6c47ce8e59402f71e657a4_prop1, etc).

**Output claims**:
- prop1
- prop2
- propN

**REST API response**:

`{ "value" : [ { "extension_<ClientId>_prop1" : "value1", "extension_<ClientId>_prop2" : "value2" }, ... ] }`

where **ClientId** comes from Query string parameter _with hyphen_ (for example, 7b230031-ff6c-47ce-8e59-402f71e657a4).

**Expected token**:
{
...
"prop1" : "value1",
"prop2" : "value2",
...
}

In other words, REST API has a response with dynamic properties without hyphen depending on query string parameter with hyphen.

Is it possible to do it? Thanks for any feedback

**Comment by JasSuri** [comment]
It’s not possible, the mappings must be static.
